src/test/java/wooteco/subway/acceptance/LineAcceptanceTest.java

```diff
@@ -0,0 +1,238 @@
+package wooteco.subway.acceptance;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import wooteco.subway.domain.Line;
+import wooteco.subway.dto.LineResponse;
+
+public class LineAcceptanceTest extends AcceptanceTest {
+
+    public static ObjectMapper om = new ObjectMapper();
+
+    static {
+        om.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+        om.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
+    }
```

Comment:
```suggestion
    @Autowired
    private ObjectMapper om;
```

이렇게도 bean으로 가져와서 사용이 가능한데, 혹시 위와 같이 사용한 이유가 있나요?

Reply:
API의 명세처럼 null인 부분을 제외하고 Request를 보내서 테스트하기 위해서 
```
static {
        om.setSerializationInclusion(JsonInclude.Include.NON_NULL);
        om.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
    }
```
해당 코드가 필요할 것이라고 생각해서 이러한 방식으로 사용했었는데, ObjectMapper를 주입 받는 방식으로 변경하고
```
  jackson:
    default-property-inclusion: non_null
```
을 application.yml 파일에 추가해주어 해결해주었습니다.